### PR TITLE
[issue-79] Added an option to not set default value in input field an…

### PR DESCRIPTION
**Pull request by palsingh**
Added an option to not set default value in input field and also updated same in README. Fixed indentation thing.

### CODE REVIEW

**Comment by palsingh** [comment]
@45kb fixed indentation

**Comment by 45kb** [comment]
@palsingh  thank you check also the if() statment so i can merge it :+1:

**Comment by palsingh** [comment]
@45kb  As I replied yo your initial comment... "With '@' in scope, we are going to get a string. So both 'true' and 'false' would be true value. That is why I used string comparison. I hope it make sense :)"

**Comment by 45kb** [comment]
@palsingh  if you don't set the attribute it is false... usually... you don't need to use attr="false"

File: src/js/angular-datepicker.js [hunk 1]
```diff
@@ -112,8 +113,10 @@
             $scope.monthNumber = Number($filter('date')(date, 'MM')); // 01-12 like
             $scope.day = Number($filter('date')(date, 'dd')); //01-31 like
             $scope.year = Number($filter('date')(date, 'yyyy'));//2014 like
-						$scope.setDaysInMonth($scope.monthNumber, $scope.year);
-            $scope.setInputValue();
+            $scope.setDaysInMonth($scope.monthNumber, $scope.year);
+            if ($scope.hideDefaultDate != 'true') {
```
Comment (45kb) [inline src/js/angular-datepicker.js]: actually should be ok just ```if ($scope.hideDefaultDate) {}``` try it ;)

**Comment by palsingh** [comment]
@45kb Not setting attribute gives undefined, but if I set it to false, it is string "false" which act as true and shows data in input field.

**Comment by 45kb** [comment]
@palsingh  ok just use ```!==`` instead of ```!=``` and i will merge it, thank you a lot for your time!

**Comment by 45kb** [comment]
thank you, will release a new version as soon as possible

**Comment by palsingh** [comment]
@45kb Done :)
Your datepicker is cool, helped a lot. Cheers.

Just to explain further why I did this, I'm using this datepicker to filter some date in a range of date. And those fields have binding with angular filter, so if default date is set, it also triggers filter and if I don't set default date, datepicker opens with max date selected. So now, date won't be set in input field and no filter will trigger on init :)

**Comment by 45kb** [comment]
Yes, for sure, was needed! You're welcome anytime, feel free to PR when you have something new!

**Comment by 45kb** [comment]
shipped https://github.com/720kb/angular-datepicker/releases/tag/0.2.10


hide-default-date is now date-set-hidden , just that, thank you !

**Pull request merged**